docs(fix): replace outdated links MTA-1772 (#416)
scaleway/docs-content: tutorials/create-esxi-cluster-dedibox/index.mdx

<Message type="important">

> To proceed with these steps, you require at least one [failover IP](/dedibox-network/ip-failover/quickstart/) with an associated [virtual MAC address](/dedibox-network/ip-failover/how-to/create-virtual-mac/). If you have not yet configured it, go to your [Dedibox Console](https://console.online.net) before proceeding.

Reviewer: ```suggestion
To proceed with these steps, you need to configure at least one [failover IP](/dedibox-network/ip-failover/quickstart/) with an associated [virtual MAC address](/dedibox-network/ip-failover/how-to/create-virtual-mac/). If you have not yet configured it, go to your [Dedibox Console](https://console.online.net) before proceeding.
```

The virtual machine retains its network identity and connections, ensuring a seamless migration process which completes in several seconds. It uses the RPN connection between your servers to avoid any impact on the public Internet connection, however some restrictions apply:

- The origin and destination machine must be in the same RPN group.
- Both servers must be connected to the same RPN-SAN and the VM uses the RPN-SAN as datastore.

> - You have ticket the **vMotion** checkbox on the RPN VMkernel network interface on both machines.

Reviewer: ```suggestion
- You have ticked the **vMotion** checkbox on the RPN VMkernel network interface on both machines.
```

> - No virtual CD/DVD drives are connected or any local datastore is in use on the VM.

Reviewer: ```suggestion
- No virtual CD/DVD drives are connected and no local datastore is being used on the VM.
```
